doc/adr/0002/graphql-server-prototype/test

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Assumes [1] below, which assumes a running trace-server; based on [2].
+# POST is used for the curl request, but GET [3] can be used although less readable.
+# Any combinations of the supported fields can be queried, which is why GraphQL.
```

Comment:
Did something get cut out from the end of the sentence? Was it meant to be something like "... which is why GraphQL was suggested as a solution."?

Reply:
No, not really; this way of saying does exist in English, AFAIK.